README.md

```diff
@@ -113,6 +113,19 @@ ee.on('error', function (err) {
 
 ```
 
+You can also pass in the specification directly:
```

Comment:
How about:

> You can also pass in the specification directly instead of reading it from a file. `basePath` should be a path which will be used to resolve other paths in the specification.